### PR TITLE
Add StatusCrowdsourceUsers request specs

### PR DESCRIPTION
**Pull request by hugopeixoto**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -14,7 +14,6 @@
 #  app_uuid               :string
 #  last_post              :datetime
 #  role                   :integer          default("0")
-#  store_owner_code       :string
```
Comment (simaob) [inline app/models/user.rb:17]: where did the store_owner_code go?

Comment (simaob) [reply app/models/user.rb]: your db version might not be up to date, I brought back the `store_owner_code`

Comment (hugopeixoto) [reply app/models/user.rb]: :facepalm: `annotate` requires me to run `db:migrate`. it does not read from schema.rb. I'll make a new PR adding it back.

Comment (simaob) [reply app/models/user.rb]: I added it =) tis cool

**Pull request merged**